nx.json

```diff
@@ -215,6 +187,19 @@
         "targetName": "pw-e2e",
         "ciTargetName": "e2e-ci"
       }
+    },
+    {
+      "plugin": "@nx/eslint/plugin",
+      "options": {
+        "targetName": "lint"
+      }
+    },
+    {
+      "plugin": "@nx/jest/plugin",
+      "include": ["packages/**/*", "graph/**/*", "nx-dev/**/*", "tools/**/*"],
```

Comment:
I would just include everything except for `e2e/**/*`

Reply:
I've tried this but it doesn't work in exclude: ` "packages/**/__fixtures__/**/*"`